src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java

```diff
@@ -1766,6 +1767,42 @@ public void siVerificaCheLaNotificaVieneRifiutata(String dpFile) {
         }
     }
 
+    @And("Si ottiene il codice IUN dalla notifica creata")
+    public void ottieniIUNdaRichiestaNotifica() {
+        piattaformaNotifichePage.setNetWorkInfos(webDriverManager.getNetworkInfosThread().get());
+        piattaformaNotifichePage.setWebDriverManager(webDriverManager);
+        piattaformaNotifichePage.setRestNotificationParam(restNotification);
+        piattaformaNotifichePage.setNotificationSingletonParam(notificationSingleton);
+        piattaformaNotifichePage.setHooksNew(hooksNew);
+        try {
+            TimeUnit.SECONDS.sleep(5);
+        } catch (InterruptedException e) {
+            throw new RuntimeException(e);
+        }
+        String urlNotificationRequest = webDriverConfig.getBaseUrl() + "notifications/sent";
+        String notificationRequestId = getNotificationRequestId(urlNotificationRequest);
+        if (notificationRequestId == null) {
+            Assertions.fail("NotificationRequestId non trovato, il codice della risposta al url " + urlNotificationRequest + " è diverso da 202 ");
```

Comment:
Qui è possibile utilizzare direttamente un assertNotNull senza ricorrere al if
```suggestion
        Assertions.assertNotNull(notificationRequestId,
        "NotificationRequestId non trovato, il codice della risposta al url " + urlNotificationRequest + " è diverso da 202"
);
```

Reply:
Grazie dell'appunto, lo integro e committo il cambio